src/dsp_tools/commands/excel2json/project.py

```diff
@@ -59,7 +59,7 @@ def excel2json(
 def _validate_folder_structure_get_filenames(data_model_files: str) -> tuple[list[Path], list[Path]]:
     if not Path(data_model_files).is_dir():
         raise UserError(f"ERROR: {data_model_files} is not a directory.")
-    folder = [x for x in Path(data_model_files).glob("*") if _non_hidden(x)]
+    folder = [x for x in Path(data_model_files).glob("*") if _non_hidden(x) and x.is_dir()]
```

Comment:
This list contains all folders, i.e. the list folder + the onto folders. So it should be called "list_folder_and_onto_folders", or it that's too long, "folders".
It should then also be changed on these lines:
- 80
- 139
- 150

The current name has now really hindered me to understand what is going on 😅